fe/fe-core/src/main/java/com/starrocks/lake/CommitRateLimiter.java

```diff
@@ -143,13 +154,14 @@ private double getPartitionCompactionScore(@NotNull Long partitionId) {
         return compactionScore != null ? compactionScore.getMax() : 0;
     }
 
-    private boolean anyCompactionScoreExceedsUpperBound(@NotNull Set<Long> partitionIds, long upperBound) {
+    // Returns the first partition id and its compaction score that exceeds the upper bound
+    private Pair<Long, Double> anyCompactionScoreExceedsUpperBound(@NotNull Set<Long> partitionIds, long upperBound) {
```

Comment:
better to use `Optional<Pair<Long, Double>>`

Reply:
done